src/index.js

```diff
@@ -16,6 +16,14 @@ import filterDirectory from './utils/filter-directory';
 
 const logger = _logger('broccoli-dependency-funnel');
 
+heimdall.registerMonitor('broccoli-dependency-funnel', function BroccoliDependencyFunnelSchema() {
```

Comment:
I think we can make this the nodes schema instead of a monitor. AFAICT this monitor will only be used here...